### PR TITLE
Add CreatePhysicalInterface to Device.

### PR DESCRIPTION
**Pull request by howbazaar**
Even though it is possible to create a physical interface for a machine, you would always do it on a device.

### CODE REVIEW

File: device.go [hunk 1]
```diff
@@ -49,6 +51,82 @@ func (d *device) Zone() Zone {
...
+		}
+		return nil, NewUnexpectedError(err)
+	}
+
+	iface, err := readInterface(d.controller.apiVersion, result)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	//	m.interfaceSet = append(m.interfaceSet, iface)
```
Comment (mjs) [inline device.go]: delete? or is it supposed to be uncommented?

**Comment by mjs** [comment]
Just one thing but LGTM otherwise.

File: device.go (hunk 1 above)
Comment (howbazaar) [inline device.go]: Should be deleted... for now. There is a maas bug where the devices isn't returning an interface set when it should be.

**Comment by howbazaar** [comment]
$$merge$$

**Pull request merged**